fuse-connect/build.gradle

```diff
@@ -3,6 +3,7 @@ plugins {
     id 'org.jetbrains.kotlin.android'
     id 'kotlin-parcelize'
     id 'maven-publish'
+    id 'com.google.android.libraries.mapsplatform.secrets-gradle-plugin'
```

Comment:
For secrets management
https://developers.google.com/maps/documentation/android-sdk/secrets-gradle-plugin